exquery-restxq-specification/restxq-1.0-specification.html

```diff
@@ -479,8 +479,8 @@ <h4>Resource Function Parameters</h4>
               The cardinality MUST allow for zero or many atomic values in the case of
               <a href="#query-param-annotation">Query</a>,
               <a href="#form-param-annotation">Form</a> or
-              <a href="#header-param-annotation">Header</a> parameters, or zero or one
-              atomic value in the case of <a href="#cookie-param-annotation">Cookie</a>
+              <a href="#form-header-annotation">Header</a> parameters, or zero or one
+              atomic value in the case of <a href="#form-cookie-annotation">Cookie</a>
```

Comment:
There is no such thing as a "Form Cookie Annotation" in RESTXQ. This was correct and should indeed be `cookie-param-annotation` - it is the other end of the link that needs to be fixed please.